kvdb/postgres/readwrite_tx.go

```diff
@@ -48,10 +52,39 @@ func newReadWriteTx(db *db, readOnly bool) (*readWriteTx, error) {
 	}, nil
 }
 
-// rooBucket is a helper function to return the always present
-// pseudo root bucket.
-func rootBucket(tx *readWriteTx) *readWriteBucket {
-	return newReadWriteBucket(tx, "kv", nil)
+func hasSpecialChars(s string) bool {
+	for _, b := range s {
+		if !(b >= 'a' && b <= 'z') &&
+			!(b >= '0' && b <= '9') && b != '-' {
```

Comment:
Add underscore too? I'm pretty sure we use that somewhere...

Reply:
It would block the reverse migration, because we don't know anymore whether _ needs to go back to _ or -

Reply:
Ah, got it.